zeus21/sfrd.py

```diff
@@ -5,14 +5,23 @@
 Author: Julian B. Muñoz
 UT Austin and Harvard CfA - January 2023
 
+Edited by Hector Afonso G. Cruz
+JHU - July 2024
 """
 
 from . import cosmology
 from .xrays import Xray_class, sigma_HI, sigma_HeI
 from . import constants
 
 import numpy as np
+import astropy
+from astropy import units as u
```

Comment:
It might be easiest to fold in astropy units into ours in a future version; we have things like ergToK, KtoeV, and freqLyA in constants.py that worry me a bit because most of the astro world just uses astropy for these quantities.